dev/VSIX/ProjectTemplates/Desktop/CSharp/ClassLibrary/Class1.cs

```diff
@@ -7,9 +7,8 @@
 // To learn more about WinUI, the WinUI project structure,
 // and more about our project templates, see: http://aka.ms/winui-project-info.
 
-namespace $safeprojectname$
+namespace $safeprojectname$;
+
+public class Class1
```

Comment:
```suggestion
public partial class Class1
```
To aid with source generators becoming more common, and potential less steps for if it needs AOT.

Reply:
`partial` added